sdk/python/v1beta1/kubeflow/katib/api/katib_client.py

```diff
@@ -259,6 +286,24 @@ def tune(
             RuntimeError: Failed to create Katib Experiment.
         """
 
+        print(
+            "Thank you for using `tune` API for LLMs hyperparameters optimization. This feature is in alpha stage Kubeflow community is looking for your feedback. Please share your experience via #kubeflow-katib Slack channel or Kubeflow Katib GitHub."
+        )
+        
+        if (
+            ((model_provider_parameters is not None) and (dataset_provider_parameters is not None)) == (objective is not None)
+        ):
```

Comment:
I think, we should validate `base_image` also.
What do you think about this:
```py
 if
 (model_provider_parameters is not None or dataset_provider_parameters is not None or trainer_parameters is not None) and
 (objective is not None or base_image is not None or parameters is not None):
```

Reply:
@andreyvelich Yes, I agree. I think we need to validate two things:
1) Ensure that only one option is chosen.
2) Verify that all necessary parameters are provided for the selected option.

We can revise the validation for the first point according to your suggestions and add validation for the second point under each option. Does this sound okay to you?

Reply:
@andreyvelich I found that `base_image` is set to "constants.BASE_IMAGE_TENSORFLOW" by default in the original API. Therefore, we cannot validate `base_image` here unless we set its default value to "None". However, I'm concerned this might cause unexpected changes for existing users. So, I have adjusted the validation here as follows:
```python
if
 (model_provider_parameters is not None or dataset_provider_parameters is not None or trainer_parameters is not None) and
 (objective is not None or parameters is not None):
```
And I have added the validation for `base_image` under the custom objective function option:
```python
if objective is not None:
    if not base_image or not parameters:
        raise ValueError("One of the required parameters is None.")
```
How do you like this?

Reply:
Sure, it looks good.